operators/awx-operator/2.5.1/metadata/annotations.yaml

```
@@ -0,0 +1,14 @@
+annotations:
+  com.redhat.openshift.versions: v4.10-v4.13n  # Core bundle annotations.
```

```suggestion
  com.redhat.openshift.versions: v4.10-v4.13  # Core bundle annotations.
```

@TheRealHaoLiu I think this is why the CI is failing.

---

We have some automation that creates these PR's.  I think there is a quick to our automation when run on iOS that caused this.  I can't modify this PR, so I put up a new one here:
* https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/3194

---

@mporrato please feel free to close this PR (Hao is OOO atm, otherwise he would say the same).